UI/window-projector.cpp

```diff
@@ -375,86 +354,87 @@ void OBSProjector::OBSRenderMultiview(void *data, uint32_t cx, uint32_t cy)
 			}
 			break;
 		default: // MultiviewLayout::HORIZONTAL_TOP_8_SCENES:
 			if (i < 4) {
-				sourceX = i * quarterCX;
-				sourceY = halfCY;
+				window->sourceX = i * window->quarterCX;
+				window->sourceY = window->halfCY;
 			} else {
-				sourceX = (i - 4) * quarterCX;
-				sourceY = halfCY + quarterCY;
+				window->sourceX = (i - 4) * window->quarterCX;
+				window->sourceY = window->halfCY +
+						window->quarterCY;
 			}
 		}
-		qiX = sourceX + thickness;
-		qiY = sourceY + thickness;
+		window->qiX = window->sourceX + window->thickness;
+		window->qiY = window->sourceY + window->thickness;
 	};
 
 	auto calcPreviewProgram = [&](bool program)
 	{
 		switch (multiviewLayout) {
 		case MultiviewLayout::VERTICAL_LEFT_8_SCENES:
-			sourceX = thickness;
-			sourceY = halfCY + thickness;
-			labelX = offset;
-			labelY = halfCY * 1.85f;
+			window->sourceX = window->thickness;
+			window->sourceY = window->halfCY + window->thickness;
+			window->labelX = window->offset;
+			window->labelY = window->halfCY * 1.85f;
 			if (program) {
-				sourceY = thickness;
-				labelY = halfCY * 0.85f;
+				window->sourceY = window->thickness;
+				window->labelY = window->halfCY * 0.85f;
 			}
 			break;
 		case MultiviewLayout::VERTICAL_RIGHT_8_SCENES:
-			sourceX = halfCX + thickness;
-			sourceY = halfCY + thickness;
-			labelX = halfCX + offset;
-			labelY = halfCY * 1.85f;
+			window->sourceX = window->halfCX + window->thickness;
+			window->sourceY = window->halfCY + window->thickness;
+			window->labelX = window->halfCX + window->offset;
+			window->labelY = window->halfCY * 1.85f;
 			if (program) {
-				sourceY = thickness;
-				labelY = halfCY * 0.85f;
+				window->sourceY = window->thickness;
+				window->labelY = window->halfCY * 0.85f;
 			}
 			break;
 		case MultiviewLayout::HORIZONTAL_BOTTOM_8_SCENES:
-			sourceX = thickness;
-			sourceY = halfCY + thickness;
-			labelX = offset;
-			labelY = halfCY * 1.85f;
+			window->sourceX = window->thickness;
+			window->sourceY = window->halfCY + window->thickness;
+			window->labelX = window->offset;
+			window->labelY = window->halfCY * 1.85f;
 			if (program) {
-				sourceX += halfCX;
-				labelX += halfCX;
+				window->sourceX += window->halfCX;
+				window->labelX += window->halfCX;
 			}
 			break;
 		default: // MultiviewLayout::HORIZONTAL_TOP_8_SCENES:
-			sourceX = thickness;
-			sourceY = thickness;
-			labelX = offset;
-			labelY = halfCY * 0.85f;
+			window->sourceX = window->thickness;
+			window->sourceY = window->thickness;
+			window->labelX = window->offset;
+			window->labelY = window->halfCY * 0.85f;
 			if (program) {
-				sourceX += halfCX;
-				labelX += halfCX;
+				window->sourceX += window->halfCX;
+				window->labelX += window->halfCX;
 			}
 		}
 	};
 
-	auto paintAreaWithColor = [&](float tx, float ty, float cx, float cy,
-			uint32_t color)
+	auto paintAreaWithColor = [&](float tx, float ty, float cx,
+			float cy, uint32_t color)
```

Comment:
Noticed here that you changed the parameter placement here unnecessarily.  The "float cy" parameter wasn't particularly going past the column limit.